big-practice/src/scripts/views/customer-view.js

```diff
@@ -8,21 +8,51 @@ class CustomerView {
     this.iconCancel = document.querySelector('.icon-cancel');
     this.btnCancel = document.querySelector('.btn-secondary');
     this.formCustomer = document.querySelector('.form-customer');
+    this.inputTitle = document.querySelectorAll('.input-title');
```

Comment:
Should use `formCustomer.querySelectorAll('.input-title');` instead of `document.querySelectorAll('.input-title');` to optimize query.